.github/workflows/typos.toml

```diff
@@ -20,6 +20,8 @@ hel = "hel"
 pn = "pn"
 # typos doesn't like the EDE in TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA
 EDE = "EDE"
+# HELO is an SMTP command
+HELO = "HELO"
```

Comment:
BTW is it HELO not EHLO?

Reply:
Both. If EHLO is not supported the library falls back to HELO.